app/assets/stylesheets/bootstrap_overrides.scss

```diff
@@ -42,3 +48,55 @@ body {
     color: $light-forest-green;
     text-align: left;
  }
+.fw-normal {
+    font-weight: normal;
+}
+.inline {
+    display: inline;
+}
+.f-left {
+    float: left;
+}
+.f-right {
+    float: right;
+}
```

Comment:
It does, but their `pull-left` and `pull-right` have the `!important` attribute and it wasn't playing nicely with the `div` blocks I was messing around with at the time. I may end up getting rid of these overrides later since I'm not currently using them for anything, but I'll wait and see.